### PR TITLE
Resolve z-fighting of the bald man NPC and the nearest gold miner NPC

### PR DESCRIPTION
**Pull request by chris-calo**
If the gold miner positioned closest to the bald man is active, the bounding box has z-fighting issues. This makes it difficult to click either one. This PR resolves that.

#### Before:

https://user-images.githubusercontent.com/3837103/147791419-b6221bb2-dfc1-477e-b168-a5a53e09c1c9.mov

#### Fixed:

https://user-images.githubusercontent.com/3837103/147791426-4562ea47-f96a-4935-b15c-439c7f2c2f80.mov

### CODE REVIEW

File: src/dapp/components/farm/Gold.tsx
Comment (adamhannigan) [inline src/dapp/components/farm/Gold.tsx:72]: Will the reference of `machineState` change in this case?

File: src/dapp/components/farm/Gold.tsx
Comment (adamhannigan) [inline src/dapp/components/farm/Gold.tsx:128]: Where possible try to create a sibling css file and import that.

E.g. `Gold.css` 

In the future we are looking to use JSS stylesheets with Typescript support but for now we do it the old fashion way with CSS 😅

**Review by adamhannigan — COMMENTED** [review]
Great initiative! Just a few small bits of feedback. @chris-calo 

Splitting the CSS into a separate file should help us refactor into JSS in the future.

File: src/dapp/components/farm/Gold.tsx
Comment (chris-calo) [reply src/dapp/components/farm/Gold.tsx:72]: Oh, local linter must have caught this one. I'll resolve it.

File: src/dapp/components/farm/Gold.tsx
Comment (chris-calo) [reply src/dapp/components/farm/Gold.tsx:128]: Yeah, that makes sense @adamhannigan – can do!

**Comment by chris-calo** [comment]
@adamhannigan this has been broken out into a new `Gold.css` file, pending approval 👍

File: src/dapp/components/farm/Gold.tsx
Comment (chris-calo) [reply src/dapp/components/farm/Gold.tsx:73]: @adamhannigan I retained this, as it doesn't appear to be affected erroneously by updates to `machineState`, since it's not a deep check of the parent object. It's proper to include it as, if the reference is no longer valid the `value` child must also have changed.

**Review by Rofrtd — APPROVED** [review]
Awesome work @chris-calo! 
This looks good to me!

File: src/dapp/components/farm/Gold.tsx
Comment (spencerdezartsmith) [inline src/dapp/components/farm/Gold.tsx:135]: Rather than putting these in the object with `true` value, you can pass them as a string along with `gather-tree`. 

```
classnames('gather-tree', 'game-object', 'gold', { ... })
```

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
Great job @chris-calo. Just one change in the `classnames` structure. Once that is done I will approve and merge.

File: src/dapp/components/farm/Gold.tsx
Comment (chris-calo) [reply src/dapp/components/farm/Gold.tsx]: @spencerdezartsmith this is resolved in the latest commit! 😁

**Review by spencerdezartsmith — APPROVED** [review]
Legend!

**Pull request merged**
